RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -1522,7 +1520,7 @@
 			mainGroup = 352629F41F7C4B9100C04F2C;
 			packageReferences = (
 				2D803F6126F144830069D717 /* XCRemoteSwiftPackageReference "nimble" */,
-				2D9C5ECB26F2815E0057FC45 /* XCRemoteSwiftPackageReference "OHHTTPStubs" */,
+				2D9C5ECB26F2815E0057FC45 /* XCRemoteSwiftPackageReference "OHHTTPStubs.git" */,
 			);
```

Comment:
This seems wrong?

Reply:
this is automatic, I think it depends on the xcode version. I'll revert anyway